src/main/java/it/smartcommunitylab/aac/webauthn/controller/WebAuthnAuthenticationController.java

```diff
@@ -0,0 +1,126 @@
+package it.smartcommunitylab.aac.webauthn.controller;
+
+import java.util.LinkedHashMap;
+import java.util.Optional;
+
+import javax.validation.Valid;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.ibm.icu.impl.Pair;
```

Comment:
please don't import external classes from third party packages for trivial use

Reply:
Removed in f5dda8a98fb396dccffe3f21455aa6b931bbd3f6